scripts/main_menu/main_menu_hud.lua

```diff
@@ -7,17 +7,32 @@ end
 
 function MainMenuHudSystem:init(gameState)
     System.init(self, gameState)
-    local root = gameState:rootGUIWindow()
+    root = gameState:rootGUIWindow()
     local microbeButton = root:getChild("Background"):getChild("MainMenuInteractive"):getChild("NewGameButton")
     local microbeEditorButton = root:getChild("Background"):getChild("MainMenuInteractive"):getChild("EditorMenuButton")
     local quitButton = root:getChild("Background"):getChild("MainMenuInteractive"):getChild("ExitGameButton")
-    
+    local loadButton = root:getChild("Background"):getChild("MainMenuInteractive"):getChild("LoadGameButton")   
     microbeButton:registerEventHandler("Clicked", mainMenuMicrobeStageButtonClicked)
     microbeEditorButton:registerEventHandler("Clicked", mainMenuMicrobeEditorButtonClicked)
+    loadButton:registerEventHandler("Clicked", mainMenuLoadButtonClicked)
     quitButton:registerEventHandler("Clicked", quitButtonClicked)
 end
 
 function MainMenuHudSystem:update(renderTime, logicTime)
+    if Engine:fileExists("quick.sav") then
+	    --print("The file exists");
+        root:getChild("Background"):getChild("MainMenuInteractive"):getChild("LoadGameButton"):enable();
+    else
+		--print("the file doesnt exist");
+        root:getChild("Background"):getChild("MainMenuInteractive"):getChild("LoadGameButton"):disable();
+    end
+end
```

Comment:
and also when a file is saved we would want to update availability but yeah something like that sounds good